code-of-conduct.md

```diff
@@ -0,0 +1,165 @@
+requires immediate action, and no moderators are available, you can also report
+the message directly to Discord.
+
+Further details of specific enforcement policies are outlined in the
+Moderation section.
+
+## Moderation
+
+These are the policies for upholding our community’s standards of conduct.
+
+1. Messages, comments, nicknames or avatar images that violate the Adafruit
+CircuitPython standards of conduct, including anything hateful, hurtful,
+oppressive, or exclusionary, are not allowed.
+2. Messages, comments, nicknames or avatar images that moderators find
+inappropriate, whether listed in the code of conduct or not, are also not
+allowed.
+3. Moderators will first respond to such behavior with a warning.
+4. We have a soft "three strikes" policy - we are willing to give the user two
+more chances to change your behavior. However, how the user respond to the
+warning is important - it will determine whether or not the user receives
+further chances.
+5. If you are receptive to the warning and change your behavior, nickname
+or avatar image accordingly, you will be given another chance.
+6. If the warning is unheeded, the user will be “kicked,” i.e., kicked out of
+the communication channel to cool off.
+7. If the user returns and continues the behavior, they will be banned,
+i.e., indefinitely excluded.
+8. If the user is unreceptive or unreasonable when warned by a moderator,
+they may be banned for a first or second offense.
+9. If the behavior is an egregious violation, the user may be banned
+immediately, without warning.
+
+The goal of the standards and moderation guidelines outlined above is to build
+and maintain a respectful community. We ask that you don’t just aim to be
+"technically unimpeachable", but rather try to be your best self. In
+particular, avoid offensive or sensitive issues, particularly if they’re
+off-topic to the channel or discussion. This all too often leads to unnecessary
+fights, perceptions or instances of disrespect, and damaged community trust.
+Worse, it can drive people away from the community entirely.
+
+If someone takes issue with something you said or did, resist the urge to
+be defensive. Just stop doing what it was they complained about and apologize.
+Even if you feel you were misinterpreted or unfairly accused, chances are good
+there was something you could’ve communicated differently — remember that it’s
+your responsibility to make your fellow Adafruit CircuitPython community members
+comfortable. Everyone wants to get along and we are all here first and foremost
+because we want to talk about cool technology. You will find that people will
+be eager to assume good intent and forgive as long as you earn their trust.
+
+We value many things beyond technical expertise, including collaboration and
+supporting others within our community. Providing a positive experience for
+other community members can have a much more significant impact than simply
+providing the correct answer.
+
+## Scope
```

Comment:
I'd move this above enforcement and moderation to set context for where it applies. I think its worth talking about two different scopes as well. The scope of where the CoC applies and the scope of where enforcement applies. (An action in the arduino server that violates our code of conduct can result in a ban from the adafruit server for example.)